Source/WebCore/page/Page.cpp

```diff
@@ -1823,6 +1823,10 @@ void Page::updateRendering()
         }
     });
 
+    forEachDocument([] (Document& document) {
```

Comment:
Can we make this a RenderingUpdateStep like the others?

Reply:
Added.